.github/actions/javascript/getGraphiteString/getGraphiteString.js

```diff
@@ -0,0 +1,45 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+const fs = require('fs');
+
+const run = () => {
+    // Prefix path to the graphite metric
+    const GRAPHITE_PATH = 'bucket1.reassure';
+
+    const regressionOutput = JSON.parse(fs.readFileSync('.reassure/output.json', 'utf8'));
```

Comment:
i think it'd be good to try/catch this as `JSON.parse` might throw an error if the passed argument is not valid

Reply:
thanks! Added the error handling